tools/run_tests/run_interop_tests.py

```diff
@@ -475,8 +476,17 @@ def __str__(self):
 
 _HTTP2_TEST_CASES = ['tls', 'framing']
 
+# test cases ran against http2 bad server but from usual grpc interop clients.
+_POSITIVE_HTTP2_SERVER_TEST_CASES = { 'data_frame_padding': 'large_unary', 'no_df_padding_sanity_test': 'large_unary' }
+
+# "badserver_test_cases holds" both "bad server only" tests and the "positive
+# http2 server" tests.
 _HTTP2_BADSERVER_TEST_CASES = ['rst_after_header', 'rst_after_data', 'rst_during_data',
-                     'goaway', 'ping', 'max_streams']
+                               'goaway', 'ping', 'max_streams']
+
+# positive and negative http2 tests are in the same list since they both target the same http2 server.
+_HTTP2_BADSERVER_TEST_CASES.extend(_POSITIVE_HTTP2_SERVER_TEST_CASES.keys())
```

Comment:
In stead of adding positive tests to the negative list, we should create a new var called _HTTP2_SERVER_TEST_CASES.